From: Ben Hutchings <ben@decadent.org.uk>
DatE: Sat, 18 Jun 2016 01:35:20 +0100
Subject: migrate: Fix ABI change in 3.16.36
Forwarded: not-needed

Hide the new #include from genksyms.

---
--- a/mm/migrate.c
+++ b/mm/migrate.c
@@ -30,7 +30,9 @@
 #include <linux/mempolicy.h>
 #include <linux/vmalloc.h>
 #include <linux/security.h>
+#ifndef __GENKSYMS__
 #include <linux/backing-dev.h>
+#endif
 #include <linux/memcontrol.h>
 #include <linux/syscalls.h>
 #include <linux/hugetlb.h>
