From: Ben Hutchings <ben@ecadent.org.uk>
Date: Mon, 27 Aug 2018 02:19:59 +0100
Subject: tcp: Avoid ABI change in 4.9.116
Forwarded: not-needed

Restore the export of tcp_send_ack().

---
--- a/net/ipv4/tcp_output.c
+++ b/net/ipv4/tcp_output.c
@@ -3540,6 +3540,7 @@ void tcp_send_ack(struct sock *sk)
 {
 	__tcp_send_ack(sk, tcp_sk(sk)->rcv_nxt);
 }
+EXPORT_SYMBOL_GPL(tcp_send_ack);
 
 /* This routine sends a packet with an out of date sequence
  * number. It assumes the other end will try to ack it.
