diff --git a/security/integrity/ima/ima_appraise.c b/security/integrity/ima/ima_appraise.c
index 1bcbc12..0abbea8 100644
--- a/security/integrity/ima/ima_appraise.c
+++ b/security/integrity/ima/ima_appraise.c
@@ -306,7 +306,8 @@ void ima_update_xattr(struct integrity_iint_cache *iint, struct file *file)
 	if (rc < 0)
 		return;
 
-	ima_fix_xattr(dentry, iint);
+	if (ima_appraise & IMA_APPRAISE_FIX)
+		ima_fix_xattr(dentry, iint);
 }
 
 /**
