From: Thomas Gleixner <tglx@linutronix.de>
Date: Fri, 8 Jul 2011 20:25:16 +0200
Subject: [PATCH 322/365] Add localversion for -RT release
Origin: https://git.kernel.org/cgit/linux/kernel/git/rt/linux-stable-rt.git/commit?id=859fc300fd9ee5039544ab5eebd6c7d9581b8186

Signed-off-by: Thomas Gleixner <tglx@linutronix.de>
---
 localversion-rt | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 localversion-rt

diff --git a/localversion-rt b/localversion-rt
new file mode 100644
index 000000000000..9a218ca23053
--- /dev/null
+++ b/localversion-rt
@@ -0,0 +1 @@
+-rt23
