From: "Steven Rostedt (VMware)" <rostedt@goodmis.org>
Date: Mon, 20 Nov 2017 22:03:22 -0500
Subject: [PATCH 337/337] Linux 4.9.61-rt52 REBASE
Origin: https://git.kernel.org/cgit/linux/kernel/git/rt/linux-stable-rt.git/commit?id=4e1998e087ce6be1ee0ed6f99d9bbba0bcd7e7ae

---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 9a218ca23053..66a5ed8bf3d7 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt23
+-rt52
