From e60cee25b5c47cb6daae970badf3d33fa878cebb Mon Sep 17 00:00:00 2001
From: Julia Cartwright <julia@ni.com>
Date: Sun, 5 Aug 2018 23:38:08 -0500
Subject: [PATCH 349/349] Linux 4.9.115-rt93 REBASE
Origin: https://www.kernel.org/pub/linux/kernel/projects/rt/4.9/older/patches-4.9.115-rt93.tar.xz

---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 9a218ca23053..e98a1fe050bd 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt23
+-rt93
-- 
2.18.0

