From 029c76fde7b2d53a91c319e7ee5e8ab1a75c05d0 Mon Sep 17 00:00:00 2001
From: Julia Cartwright <julia@ni.com>
Date: Mon, 26 Feb 2018 10:43:05 -0600
Subject: [PATCH 350/350] Linux 4.9.98-rt76 REBASE
Origin: https://www.kernel.org/pub/linux/kernel/projects/rt/4.9/older/patches-4.9.98-rt76.tar.xz

---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 9a218ca23053..c1f2720eaf25 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt23
+-rt76
-- 
2.17.0

