From: "Steven Rostedt (VMware)" <rostedt@goodmis.org>
Date: Thu, 30 Nov 2017 19:12:26 -0500
Subject: [PATCH 351/351] Linux 4.9.68-rt60 REBASE
Origin: https://git.kernel.org/cgit/linux/kernel/git/rt/linux-stable-rt.git/commit?id=1ae113e318f6532b9d16c6e2f5417f37ce5fc6fb

---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 9a218ca23053..66fa05e70f29 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt23
+-rt60
