From: Julia Cartwright <julia@ni.com>
Date: Mon, 15 Jan 2018 13:10:48 -0600
Subject: [PATCH 351/351] Linux 4.9.76-rt61 REBASE
Origin: https://git.kernel.org/cgit/linux/kernel/git/rt/linux-stable-rt.git/commit?id=6f503aeecec8e5041a4dc1e17ad483b449855814

---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 9a218ca23053..9b7de9345ef4 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt23
+-rt61
-- 
2.16.1

