From: "Luis Claudio R. Goncalves" <lgoncalv@redhat.com>
Date: Sat, 30 Jan 2021 15:40:05 -0300
Subject: [PATCH 365/365] Linux 4.9.254-rt169 REBASE
Origin: https://git.kernel.org/cgit/linux/kernel/git/rt/linux-stable-rt.git/commit?id=011a82a0d661a1dc0f3fb7cad87341f8ef79a9d0

Signed-off-by: Luis Claudio R. Goncalves <lgoncalv@redhat.com>
---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 9a218ca23053..f95a74c96a45 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt23
+-rt169
