From: Clark Williams <williams@redhat.com>
Date: Mon, 22 Jun 2020 19:08:20 -0500
Subject: [PATCH 366/366] Linux 4.9.228-rt147 REBASE
Origin: https://git.kernel.org/cgit/linux/kernel/git/rt/linux-stable-rt.git/commit?id=f6a7beeaddaa97deeb455bd03070b603a1d4991f

Signed-off-by: Clark Williams <williams@redhat.com>
---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 9a218ca23053..d3fa072b6b43 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt23
+-rt147
