From c3c60b488c904971c9ff0bd40ac37d69a51c250b Mon Sep 17 00:00:00 2001
From: Clark Williams <clark.williams@gmail.com>
Date: Thu, 21 Mar 2024 11:02:35 -0500
Subject: [PATCH 64/64] Linux 6.1.82-rt27 REBASE
Origin: https://www.kernel.org/pub/linux/kernel/projects/rt/6.1/older/patches-6.1.82-rt27.tar.xz

Signed-off-by: Clark Williams <clark.williams@gmail.com>
---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index c5b71f9a229d..be1e37b64ad0 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt25
+-rt27
-- 
2.44.0

