From 65ff976c34c0c2c021d2589c8fe373bcc8585aeb Mon Sep 17 00:00:00 2001
From: "Luis Claudio R. Goncalves" <lgoncalv@redhat.com>
Date: Wed, 9 Mar 2022 18:05:40 -0300
Subject: [PATCH 318/318] Linux 5.10.104-rt63 REBASE
Origin: https://www.kernel.org/pub/linux/kernel/projects/rt/5.10/older/patches-5.10.104-rt63.tar.xz

Signed-off-by: Luis Claudio R. Goncalves <lgoncalv@redhat.com>
---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 21988f9ad53f1..b0e8dd7bd7074 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt34
+-rt63
-- 
2.35.1

