From: "Luis Claudio R. Goncalves" <lgoncalv@redhat.com>
Date: Wed, 19 Feb 2025 19:01:41 -0300
Subject: [PATCH 323/323] Linux 5.10.234-rt126 REBASE
Origin: https://git.kernel.org/cgit/linux/kernel/git/rt/linux-stable-rt.git/commit?id=93649261a6376f74525152e2e49e4d809f37e586

Signed-off-by: Luis Claudio R. Goncalves <lgoncalv@redhat.com>
---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 21988f9ad53f..a44fd6cdf249 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt34
+-rt126
