From: "Luis Claudio R. Goncalves" <lgoncalv@redhat.com>
Date: Thu, 23 Jan 2025 17:23:13 -0300
Subject: [PATCH 323/323] Linux 5.10.233-rt125 REBASE
Origin: https://git.kernel.org/cgit/linux/kernel/git/rt/linux-stable-rt.git/commit?id=99b6192744c7cbc264724efb54f053a9787a7d31

Signed-off-by: Luis Claudio R. Goncalves <lgoncalv@redhat.com>
---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 21988f9ad53f..9d0cfbcac7f4 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt34
+-rt125
