From 36101c8636ee1061aa97c3d1251ba20bb9c43617 Mon Sep 17 00:00:00 2001
From: Clark Williams <williams@redhat.com>
Date: Mon, 12 Apr 2021 15:03:55 -0500
Subject: [PATCH 335/347] Linux 4.19.185-rt76 REBASE
Origin: https://www.kernel.org/pub/linux/kernel/projects/rt/4.19/older/patches-4.19.245-rt109.tar.xz

Signed-off-by: Clark Williams <williams@redhat.com>
---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 1199ebade17b..c1f2720eaf25 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt16
+-rt76
-- 
2.36.1

