From 2cbd4f29347db80cc4de7240f1efac72a7730893 Mon Sep 17 00:00:00 2001
From: Clark Williams <williams@redhat.com>
Date: Fri, 16 Apr 2021 16:35:36 -0500
Subject: [PATCH 338/338] Linux 4.19.188-rt77 REBASE
Origin: https://www.kernel.org/pub/linux/kernel/projects/rt/4.19/older/patches-4.19.188-rt77.tar.xz

Signed-off-by: Clark Williams <williams@redhat.com>
---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index c1f2720eaf25..595841feef80 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt76
+-rt77
-- 
2.30.2

