From a6e5dcc3df24b4e3907aa04533d7d167c5210d00 Mon Sep 17 00:00:00 2001
From: Clark Williams <williams@redhat.com>
Date: Mon, 19 Jul 2021 13:46:22 -0500
Subject: [PATCH 338/338] Linux 4.19.197-rt84 REBASE
Origin: https://www.kernel.org/pub/linux/kernel/projects/rt/4.19/older/patches-4.19.197-rt84.tar.xz

Signed-off-by: Clark Williams <williams@redhat.com>
---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index c1f2720eaf25..fc6ea32352bc 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt76
+-rt84
-- 
2.31.1

