From 4cb3fd41bca3b672be5f1b89453cabe79e9245df Mon Sep 17 00:00:00 2001
From: Clark Williams <williams@redhat.com>
Date: Mon, 5 Jul 2021 13:38:27 -0500
Subject: [PATCH 338/338] Linux 4.19.196-rt83 REBASE
Origin: https://www.kernel.org/pub/linux/kernel/projects/rt/4.19/older/patches-4.19.196-rt83.tar.xz

Signed-off-by: Clark Williams <williams@redhat.com>
---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index c1f2720eaf25..6a72c0fb772f 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt76
+-rt83
-- 
2.31.1

