From ec9f512281151c06e7acd94f21f7f7bb97b14e06 Mon Sep 17 00:00:00 2001
From: Daniel Wagner <wagi@monom.org>
Date: Wed, 13 Jul 2022 16:25:15 +0200
Subject: [PATCH 342/342] Linux 4.19.252-rt112 REBASE
Origin: https://www.kernel.org/pub/linux/kernel/projects/rt/4.19/older/patches-4.19.252-rt112.tar.xz

Signed-off-by: Daniel Wagner <wagi@monom.org>
---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 1199ebade17b..0c40e2660574 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt16
+-rt112
-- 
2.37.0

