From d28ccbd42ce8c5b5d0b5f79160bfe0c5c98f9037 Mon Sep 17 00:00:00 2001
From: Daniel Wagner <wagi@monom.org>
Date: Fri, 11 Mar 2022 08:57:52 +0100
Subject: [PATCH 345/345] Linux 4.19.233-rt105 REBASE
Origin: https://www.kernel.org/pub/linux/kernel/projects/rt/4.19/older/patches-4.19.233-rt105.tar.xz

Signed-off-by: Daniel Wagner <wagi@monom.org>
---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 9ea5981fa956..b4e5b00e24e1 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt101
+-rt105
-- 
2.35.1

