From b7cf2f6b538ab97b989cbb2d10bf804994874ba5 Mon Sep 17 00:00:00 2001
From: Daniel Wagner <wagi@monom.org>
Date: Tue, 14 Jun 2022 07:47:54 +0200
Subject: [PATCH 347/347] Linux 4.19.246-rt110 REBASE
Origin: https://www.kernel.org/pub/linux/kernel/projects/rt/4.19/older/patches-4.19.246-rt110.tar.xz

Signed-off-by: Daniel Wagner <wagi@monom.org>
---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 2e0a3c960575..b3e668a8fb94 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt107
+-rt110
-- 
2.36.1

