From: Daniel Wagner <wagi@monom.org>
Date: Mon, 19 Dec 2022 09:01:51 +0100
Subject: [PATCH 352/352] Linux 4.19.269-rt119 REBASE
Origin: https://git.kernel.org/cgit/linux/kernel/git/rt/linux-stable-rt.git/commit?id=ad6133d8a05e2769cea70fc65d36e39df50d97f0

Signed-off-by: Daniel Wagner <wagi@monom.org>
---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 1199ebade17b..ba6e8821d159 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt16
+-rt119
